core/block_validator.go

```diff
@@ -123,22 +131,29 @@ func (v *BlockValidator) ValidateState(block *types.Block, statedb *state.StateD
 			}
 		}
 	}
+	time6 := common.PrettyDuration(time.Since(start))
```

Comment:
IMO it would be more useful to print time deltas, instead of time to each checkpoint. Something like:
```
t0 := time.now() // start
...
t1 := time.now()
...
t2 := time.now()
...
t3 := time.now()
...
t_stop := time.now() // stop 
log.Info("times during validate state:",
    "t0->t1=", common.PrettyDuration(t1-t0),
    "t1->t2=", common.PrettyDuration(t2-t1),
    "t2->t3=", common.PrettyDuration(t3-t2),
    "total=", common.PrettyDuration(t_stop-t_3),
```

Reply:
Every time I look at these logs, I am scanning all the numbers to see which one consumed the most of the total append time. Would be easier to just see which number is bigger